lib/vector-api-client/graphql/queries/components.graphql

```diff
@@ -5,5 +5,8 @@ query ComponentsQuery {
     eventsProcessedTotal {
       eventsProcessedTotal
     }
+    bytesProcessedTotal {
```

Comment:
Heads up: there's been some significant renaming work in VectorMetricsVille. These metrics are now prefixed with `vector_` on the Prometheus side of things. I'm not sure if that entails that these should be renamed, but wanted to let you know.